template/utils/test-utils.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FunctionComponent } from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
+import { configureAxe } from 'jest-axe';
+import { ThemeProvider } from 'emotion-theming';
+import {
+  render as renderTest,
+  waitFor,
+  act,
+  cleanup,
+  RenderResult,
+} from '@testing-library/react';
+import { renderHook, act as actHook } from '@testing-library/react-hooks';
+import userEvent from '@testing-library/user-event';
+import { light } from '@sumup/design-tokens';
+
+export type RenderFn<T = any> = (
+  component: React.ReactElement,
+  ...rest: any
+) => T;
+
+const WithProviders: FunctionComponent = ({ children }) => (
+  <ThemeProvider theme={light}>{children}</ThemeProvider>
+);
+
+export const render: RenderFn<RenderResult> = (component, options) =>
+  renderTest(component, { wrapper: WithProviders, ...options });
+export const renderToHtml: RenderFn<string> = (component) =>
+  renderToStaticMarkup(<WithProviders>{component}</WithProviders>);
+export const create = (...args: Parameters<RenderFn<RenderResult>>) => {
+  const { container } = render(...args);
+  return container.children.length > 1
+    ? container.children
+    : container.firstChild;
+};
```

Comment:
Would you also want to take an approach where we only have one render function and adjust the Foundry template instead?

Reply:
This seems like two separate questions to me:

> Would you also want to [...] adjust the Foundry template instead?

Yes, I've moved the `test-utils` file to the repo root — same as `create-sumup-next-app` — and opened a PR in Foundry to update the template (https://github.com/sumup-oss/foundry/pull/163).

> Would you also want to take an approach where we only have one render function?

What do you mean by "one render function"? `render`, `create`, and `renderToHtml` serve different purposes each. This starter has additional tools for accessibility testing with aXe.

Reply:
Ah, that's what I thought you confirmed in our chat on Slack the other day. It was my understanding, and that's what I did in the CRA template, that you nowadays do not need `create` and `renderToHtml` and Axe or snapshotting just works with the `container` from react testing library. Hence, you would only need to re-export the custom `render`.

Reply:
You're right. I've simplified the test utils in the latest commits.